src/components/Select/Select.tsx

```diff
@@ -1,27 +1,20 @@
-import React, { FC, useState, useEffect, CSSProperties } from "react";
-import styled from "styled-components";
-import Select, { components, IndicatorProps, ValueType } from "react-select";
-import { colors } from "../../styles/colors";
+import React, { Fragment } from "react";
+import Select, { components, IndicatorProps } from "react-select";
 import { CaretIcon } from "../Icon/Icons";
-import tickSVG from "../../assets/tick.svg";
-import { isMobile } from "@utils";
-import { throttle } from "lodash";
-import { SelectNative } from "./SelectNative";
-import { SelectComponentProps, OptionType, WidthProps, SizeProps, DisabledUIProps } from "./SelectTypes";
-import { Spacer } from "../../styled";
+import { hoc } from "@utils";
+import { SelectNative } from "@components/Select/SelectNative";
```

Comment:
change to local reference path ( ./SelectNative ). Do not reference something local with absolute imports